197.js

```diff
@@ -0,0 +1,32 @@
+//https://code.mu/ru/javascript/book/prime/inbuilt/array/searching/
+let str197 = [1, 2, 3, 4, 5];
+if(str197.includes(3)){
+    console.log('yes')
+} else {
+    console.log('no')
+}
+
+let str1972 = [1, 2, 3, 4, 5];
+let res = str1972.find(function(elem1972){
+   return elem1972 === 3
+})
+if(res){
+    console.log('yes')
+} else {
+    console.log('no')
+}
```

Comment:
усе 4 варыянты напісаны добра

вось тут для `.find` можна ужываць стрэлачную функцыю, каб код  выглядаў прасцей
```suggestion
let str1972 = [1, 2, 3, 4, 5];
if (str1972.find(elem => elem === 3)) {
  console.log('yes')
} else {
  console.log('no')
}
```
Я памятаю, што ты яшчэ добра не працаваў са стрэлачнымі функцыямі. Я хачу, каб ты прызвычайваўся да іх па крыху

Reply:
Усё  супер :+1: 
Фіксі канфлікт і мержы

Reply:
А паспрабуй перапісаць гэты `if / else` блок на [тернарны аператар](https://code.mu/ru/javascript/book/prime/conditions/ternary-operator/), каб атрымаўся адзін радок